tests/ssh_test.py

```diff
@@ -1,10 +1,9 @@
-
-import unittest
-
 import mitogen
 import mitogen.ssh
 import mitogen.utils
 
+import unittest2 as unittest
```

Comment:
This looks amazing, but please, no import aliases, it makes code impossible to dive into. If you're using unittest2 please name it explicitly throughout the code, otherwise confusion will abound.